packages/eslint-plugin/src/utils/splitShorthands.js

```diff
@@ -189,13 +191,33 @@ export function splitSpecificShorthands(
     return [[toCamelCase(property), isNumber ? Number(value) : value]];
   }
 
-  const longformStyle: {
-    [key: string]: number | string,
-  } = {};
+  const directionMap = {
+    left: 'inline-start',
+    right: 'inline-end',
+  };
+
+  // List directional properties for later inline transform if needed
+  const directionalProperties = [
+    'border-width',
+    'border-color',
+    'border-style',
```

Comment:
We should include `borderRadius` too. Is there anything else? Where is the configuration for enabling this for `padding` and `margin` located? Do we know if OSS users get consistent behavior across properties (it should default for physical properties for every shorthand)

Reply:
These are the supported properties, I believe it's just these that are directional:

borderColor
borderWidth
borderStyle
borderRadius
margin
padding
marginBlock
marginInline
paddingBlock
paddingInline

And these that are left as is:
borderTop
borderRight
borderBottom
borderLeft
background
font
outline

There's a global config `preferInline` that's set when configuring the lint rule; it's default false for all properties, so by default all properties use physical properties when this config is not set.